modules/UI/videolayout/SmallVideo.js

```diff
@@ -82,13 +82,14 @@ function SmallVideo(VideoLayout) {
     this.disableUpdateView = false;
 
     /**
-     * Statistics to display within the connection indicator. With new updates,
-     * only changed values are updated through assignment to a new reference.
+     * The current state of the user's bridge connection. The value should be
+     * a string as enumerated in the library's participantConnectionStatus
+     * constants.
      *
      * @private
-     * @type {object}
+     * @type {string|null}
      */
-    this._cachedConnectionStats = {};
+    this._connectionStatus = null;
```

Comment:
I used to jam connection status as part of the stats object to be passed in but I separated them because stats are getting accessed directly by ConnectionIndicator.